msprime/trees.py

```diff
@@ -2169,6 +2169,38 @@ def get_pairwise_diversity(self, samples=None):
             leaves = list(samples)
         return self._ll_tree_sequence.get_pairwise_diversity(leaves)
 
+    def get_mean_tmrca(self, leaf_sets, windows):
+        """
+        Finds the mean time to most recent common ancestor between pairs of samples
+        from each group of leaves and in each window. Returns the upper triangle
+        (including the diagonal) in row-major order, so if the output is `x`, then:
+
```

Comment:
I think this is breaking the docs build, which is breaking the CI.

Can you try running ``make`` in the docs directory and fixing the issue please?